uvicorn/protocols/websockets/wsproto_impl.py

```diff
@@ -150,6 +174,7 @@ def handle_text(self, event):
 
     def handle_bytes(self, event):
         self.bytes += event.data
+        # todo: question (AK): should we guard the size of this (hypercorn does)
```

Comment:
@tomchristie any comment on this? Maybe something to look at in another PR.

Reply:
Yes, but I'd suggest we treat that as a separate issue.